library/Faktory/Worker.hs

```diff
@@ -3,21 +3,19 @@
 -- Runs forever, @FETCH@-ing Jobs from the given Queue and handing each to your
 -- processing function.
 --
-module Faktory.Worker
-  ( WorkerHalt(..)
-  , Worker(..)
-  , runWorker
-  , runWorkerEnv
-  , quietWorker
-  , createWorker
-  , startWorker
-  , jobArg
-  )
-where
+module Faktory.Worker (
+  WorkerHalt (..),
+  WorkerConfig (isQuieted, client, workerId, workerSettings, settings),
```

Comment:
[Consider] Should this just be `(...)`?  Why export all these individually?  Looks like this is just all of them.